gcloud-java-compute/src/main/java/com/google/gcloud/compute/StorageImageConfiguration.java

```diff
@@ -27,14 +27,13 @@
  * A Google Compute Engine raw image configuration. This class can be used to create images from a
```

Comment:
Related to the comment above. Is raw refers to the sourceType and if so is that guarantied to be fixed?
If the answers are yes and no then when we add a new enum value to the `SourceType` we would also need
to cleanup the javadoc...